test/testpy/__init__.py

```diff
@@ -53,11 +53,15 @@ def AfterRun(self, result):
     # delete the whole tmp dir
     try:
       rmtree(self.tmpdir)
+      if "NODE_PIPE_DIR" in os.environ:
+        rmtree(os.environ["NODE_PIPE_DIR"]);
```

Comment:
One alternative if we are worried about deleting the contents of the directory is to just delete the files we create.  If we think that would be ok I can try that out.